cluster/src/dunit/scala/io/snappydata/cluster/QueryRoutingDUnitTest.scala

```diff
@@ -29,6 +29,7 @@ import com.gemstone.gemfire.distributed.internal.membership.InternalDistributedM
 import com.gemstone.gemfire.internal.cache.PartitionedRegion
 import com.pivotal.gemfirexd.internal.engine.Misc
 import com.pivotal.gemfirexd.internal.engine.distributed.utils.GemFireXDUtils
+import com.pivotal.gemfirexd.internal.shared.common.reference.SQLState
```

Comment:
Since there is no change in the file, this import can be removed

Reply:
Removed unused import